homeassistant/components/hive/__init__.py

```diff
@@ -69,6 +70,20 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
                 },
             )
         )
+        async_create_issue(
```

Comment:
What if the import fails? See ADR-0021

Reply:
@frenck can you take a look at the new update here and see if you think it's a feasible way to deal with adr 0021?